### ISSUE TITLE
Library showcase page.

### ISSUE THREAD

**Issue by agpar** [DDMAL/misirlou#94]
I wonder if you guys think it would be a good idea to have a 'library showcase' page. Now that the text under the search bars actually reads '... from X libraries', it got me thinking that it might be nice to have a page where we can display library logos, links to search for a specific libraries stuff, and to visit the libraries website. It could be basically like the front page (4-wide grid of logos with links). Reasons I think this would be good:
- We should do as much as we can to give credit and gratitude to the libraries that are scanning and serving the documents we display.
- We currently reference the fact that we are sourcing 'from X libraries', but give no easy way to figure out what those libraries are, what stuff we have from those libraries, or how to find those libraries.
- We _will_ have a good idea which libraries we have imported stuff from at any one time. Unless something shocking happens with respect to how manifests are categorized and organized by libraries, we will be left manually searching for music documents to import. Therefore, we have no excuse about there being too many libraries or the process being too automated to properly reference all our sources.

Thoughts?

**Comment by ahankinson** [comment]
Yes! I like this idea.

I'm actually not a _huge_ fan of the 3x3 tiles, but they are good for eye candy. I think we should have some sort of 'browse' interface, but perhaps a list more like the search results, where we can facet by provider.

**Comment by jeromepl** [comment]
Good idea! Personally I would see it more like Alex's idea: as a grid. Here is something similar from google for inspiration: https://summerofcode.withgoogle.com/organizations/?sp-page=4

**Comment by agpar** [comment]
Forgot to mention this whenever it happened, but we've had Sources for a while, and each Manifest is either assigned to or creates a new Source when it's imported. You can have a look at them on the production site, even http://musiclibs.net/sources

All that's really missing is a fancy front end and a view to filter manifests by a particular Source.
